util/import/BUILD.bazel

```diff
@@ -1,11 +1,17 @@
 load("//rust:defs.bzl", "rust_library", "rust_proc_macro", "rust_test")
+load("//rust/private:transitions.bzl", "with_import_macro_bootstrapping_mode")
```

Comment:
You can use `# buildifier: disable=bzl-visibility` to disable buildifier warning

Reply:
Done